commcare_connect/audit/models.py

```
@@ -0,0 +1,66 @@
+    opportunity_access = models.ForeignKey(
+        OpportunityAccess, on_delete=models.CASCADE, related_name="audit_report_entries"
+    )
+    results = models.JSONField(default=dict)
+    flagged = models.BooleanField(default=False)
+    reviewed = models.BooleanField(default=False)
+    review_action = models.CharField(max_length=32, choices=ReviewAction.choices, null=True, blank=True)
+
+    class Meta:
+        indexes = [
+            models.Index(fields=["audit_report", "flagged", "reviewed"]),
+        ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)